pkg/ipamfree/calico.go

```diff
@@ -0,0 +1,29 @@
+package ipamfree
+
+import (
+    "fmt"
+    "log"
+    "net"
+    "os/exec"
+
+    "github.com/nokia/danm/crd/apis/danm/v1"
+    "github.com/nokia/danm/crd/client/clientset/versioned"
+)
+
+func CalicoIpamFreeCondition(danmClient versioned.Interface, ep *v1.DanmEp, dnet *v1.DanmNet) bool {
+    return ep.Spec.NetworkType == "calico"
```

Comment:
was referring to this BTW: https://github.com/nokia/danm/blob/master/user-guide.md#using-ipam-with-static-backends

Reply:
I see, I updated to use `imap.WasIPAllocatedByDanm`